doc/nrf/known_issues.rst

```diff
@@ -1365,6 +1365,18 @@ NCSDK-16644: :ref:`caf_sensor_manager` macro incorrectly converts float to senso
 
   **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``7e8c23a6632632f0cee885abe955e37a6942911d``).
 
+.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+NCSIDB-925: Event listeners in the :ref:`app_event_manager` may overlap when using a non-default naming convention
+In order to locate the event listener's start and stop address ranges, sections have to be sorted by the name with added postfixes.
+Hence, using a non-default event naming scheme may break the expected listeners sorting.
+  In this situation, one of the event listeners arrays might be placed inside the other.
+  So, when the event related to the outer listeners is processed, the event listeners that are inside are also executed.
+  To resolve this issue, a new implementation was introduced that uses a section naming that cannot be used as event name (invalid variable identifier).
```

Comment:
```suggestion
NCSIDB-925: Event listeners in the :ref:`app_event_manager` may overlap when using a non-default naming convention
  In order to locate the event listener's start and stop address ranges, sections have to be sorted by the name with added postfixes.
  Hence, using a non-default event naming scheme may break the expected listeners sorting.
  In this situation, one of the event listeners arrays might be placed inside the other.
  So, when the event related to the outer listeners is processed, the event listeners that are inside are also executed.
  To resolve this issue, a new implementation was introduced that uses a section naming that cannot be used as event name (invalid variable identifier).
```
You may get a doc build failure.

Reply:
Yes. Thanks. I had already located this issue during manual implementation of @MarekPieta comments. Already fixed (locally).